single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultDiagramLabelProvider.java

```diff
@@ -189,6 +189,22 @@ private void addBranchStatusDecorator(List<NodeDecorator> nodeDecorators, Node n
         }
     }
 
+    private void addBranchStatusDecorator(List<NodeDecorator> nodeDecorators, Node node, Direction direction, Branch<?> branch) {
```

Comment:
This can be removed, `BranchStatus` is a `Connectable` extension, despite its name. So, `addBranchStatusDecorator` has to be called on feederTypes branches/2wt only if `network.getBranch(feederNode.getEquipmentId())` returns a connectable.

But, if it is a TieLine, we should call it on the corresponding dangling line. Let's keep that for later?